scripts/deploy-cloudflare-workers.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Staging deploys
+if git diff --name-only origin/main | grep '^workers/' > /dev/null; then
+  echo "[deploy-cloudflare-workers] Change detected. Deploying to staging..."
+
+  ENV="staging"
+  DEPLOY=true
+
+# Production Deploys
+elif [[ "$CIRCLE_BRANCH" == "release" ]]; then
+  if git diff --name-only HEAD~1 | grep '^workers/' > /dev/null; then
```

Comment:
I don't think we can guarantee there's a diff in the most recent commit. As an alternative, is it OK to always deploy the worker?

Reply:
It is ok to deploy every time! I just wanted to avoid it because of the scary intersection. But its likely fine.